d2g.go

```diff
@@ -49,8 +55,7 @@ func Convert(dwPayload *dockerworker.DockerWorkerPayload) (gwPayload *genericwor
 	if err != nil {
 		return
 	}
-	var gwCommand [][]string
-	gwCommand, err = command(dwPayload, dwImage, gwArtifacts, gwWritableDirectoryCaches)
+	gwPayload.Command, err = command(dwPayload, dwImage, gwPayload.Artifacts, gwWritableDirectoryCaches)
```

Comment:
I assumed here that there wouldn't ever be a default command, so I didn't create a setter for this one. Lmk if you think I should.